netconf/rpc.go

```diff
@@ -78,3 +79,8 @@ func MethodUnlock(target string) RawMethod {
 func MethodGetConfig(source string) RawMethod {
 	return RawMethod(fmt.Sprintf("<get-config><source><%s/></source></get-config>", source))
 }
+
+// RawRPC allows any RPC command to be run on the target.
+func RawRPC(command string) RawMethod {
+	return RawMethod(fmt.Sprintf("%s", command))
```

Comment:
Why are you using fmt.Sprintf() here? _command_ can only ever be a string and you're not doing any variable interpolation.